Mathlib/Data/Finset/RangeAdd.lean

```diff
@@ -0,0 +1,34 @@
+/-
+Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura, Jeremy Avigad, Minchao Wu, Mario Carneiro
+-/
+import Mathlib.Algebra.Group.Embedding
+import Mathlib.Algebra.Group.Nat.Defs
+import Mathlib.Data.Finset.Image
+
+/-!
+# `Finset.range` and addition of natural numbers
```

Comment:
The contents of this file appear to only be used in `Data.Nat.Count` but I don't think that's a good place to move them to. So let's keep a separate file.